get-resource.sh

```diff
@@ -26,8 +26,10 @@ IPA_BASEURI="${IPA_BASEURI:-https://tarballs.opendev.org/openstack/ironic-python
 IPA_BRANCH="$(echo "${IPA_BRANCH:-master}" | tr / -)"
 IPA_FLAVOR="${IPA_FLAVOR:-centos9}"
 
-FILENAME="ipa-${IPA_FLAVOR}-${IPA_BRANCH}"
-FILENAME_EXT=".tar.gz"
+FILENAME="${FILENAME:-ipa-${IPA_FLAVOR}-${IPA_BRANCH}}"
```

Comment:
I wonder if it's going to be confusing that FILENAME does not include the extension. Maybe change the code to accept a full FILENAME and strip the extension?

Reply:
Okay I will change I was debating that myself too.